packages/docsearch-css/src/button.css

```
@@ -43,6 +43,7 @@
 
 .DocSearch-Button-Keys {
   display: flex;
+  width: calc(2 * 20px + 2 * 0.4em);
```

I think we should keep it loose for people using bigger fonts for instance.

```suggestion
  min-width: calc(2 * 20px + 2 * 0.4em);
```

---

Indeed, updated in https://github.com/algolia/docsearch/pull/1033/commits/bfa436a97e03f26310e9989847e1792c5f5a61ef